examples/basic.py

```diff
@@ -27,13 +31,12 @@
 
 for key in env['featuresMap']:
     params = {"featureKey": key}
```

Comment:
These parameters should be changed for "decide" call. It looks like you're still using old 'activate' params.
We should confirm all these sample codes run ok,